docs/getting-started/first-source.md

```diff
@@ -1,14 +1,14 @@
-In this tutorial, you will be using the [CloudEvents Player](https://github.com/ruromero/cloudevents-player) to showcase the core concepts of Knative Eventing. By the end of this tutorial, you should have something that looks like this:
+In this tutorial, you will be using the [CloudEvents Player](https://github.com/ruromero/cloudevents-player){target=blank} to showcase the core concepts of Knative Eventing. By the end of this tutorial, you should have something that looks like this:
```

Comment:
In this tutorial, you use the [CloudEvents Player](https://github.com/ruromero/cloudevents-player){target=blank} to showcase the core concepts of Knative Eventing. By the end of this tutorial, you will have something that looks like this: